webserver/pkgpkr/static/styles.css

```
@@ -244,6 +244,15 @@ body {
   font-size: 14px;
 }
 
+.tooltip-left .tooltiptext-left a {
```

You can avoid duplicating some of this CSS with the following syntax:

```suggestion
.tooltip .tooltiptext a,
.tooltip-left .tooltiptext-left a {
```

---

Just make sure to delete the tooltip styling above after you do this. 😄